lms/djangoapps/course_api/serializers.py

```diff
@@ -33,9 +35,9 @@ class _CourseApiMediaCollectionSerializer(serializers.Serializer):  # pylint: di
     course_video = _MediaSerializer(source='*', uri_attribute='course_video_url')
 
 
-class CourseSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+class CourseListSerializer(serializers.Serializer):  # pylint: disable=abstract-method
```

Comment:
`CourseList` is the name used in the view: [CourseListView](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_api/views.py#L113).

Essentially, we now have a different serializer for each view.  One for the Detail View and one of the List View.

The Detail View returns a single Course object, while the List View returns a paginated list of Course objects.